debian/ubuntu-advantage-tools.postinst

```diff
@@ -368,6 +371,15 @@ remove_old_systemd_units() {
     fi
 }
 
+create_public_machine_token_file() {
+    /usr/bin/python3 -c "
```

Comment:
maybe add a comment here explaining that the MachineTokenFile class handles redacting secrets from the private file and writing the redacted version to the public file